flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -195,8 +201,15 @@ public FlinkInputFormat buildFormat() {
       } else {
         contextBuilder.project(FlinkSchemaUtil.convert(icebergSchema, projectedSchema));
       }
-
-      return new FlinkInputFormat(tableLoader, icebergSchema, io, encryption, contextBuilder.build());
+      localityPreferred = localityEnabled();
+
+      return new FlinkInputFormat(
```

Comment:
We don't typically put each argument on a separate line. Instead, we just put as many as we can on each line.

Reply:
got it.